docs/src/main/sphinx/installation/containers.rst

```diff
@@ -0,0 +1,104 @@
+===================
+Trino in containers
+===================
+
+The Trino project provides `a Docker image <https://hub.docker.com/r/trinodb/trino>`_
+that includes the Trino server and a default configuration.
+
+Running the container
+---------------------
+
+To run Trino in Docker, you must have the Docker engine installed on your
+machine. You can download Docker from the `Docker website <https://www.docker.com>`_,
+or use the packaging system of your operating systems.
+
+Use the ``docker`` command to create a container from the ``trinodb/trino``
+image. Assign it the ``trino`` name, to make it easier to reference it later.
+Run it in the background, and map the default Trino port, which is 8080,
+from inside the container to port 8080 on your workstation.
+
+.. code-block:: text
+
+    docker run --name trino -d -p 8080:8080 trinodb/trino
+
+Without specifying the container image tag, it defaults to ``latest``,
+but a number of any released Trino version can be used, for example
+``trinodb/trino:390``.
+
+Run ``docker ps`` to see all the containers running in the background.
+
+.. code-block:: text
+
+    % docker ps
+    CONTAINER ID   IMAGE               COMMAND                  CREATED        STATUS                  PORTS                    NAMES
+    955c3b3d3d0a   trinodb/trino:390   "/usr/lib/trino/bin/…"   39 hours ago   Up 39 hours (healthy)   0.0.0.0:8080->8080/tcp   trino
+
+Executing queries
+-----------------
+
+The image includes the Trino command-line interface (CLI) client, ``trino``.
+Execute it in the existing container to connect to the Trino server running
+inside it. After starting the client, type and execute a query on a table
+of the ``tpch`` catalog, which includes example data:
+
+.. code-block:: text
+
+    $ docker exec -it trino trino
+    trino> select count(*) from tpch.sf1.nation;
+     _col0
+    -------
+        25
+    (1 row)
+
+    Query 20181105_001601_00002_e6r6y, FINISHED, 1 node
+    Splits: 21 total, 21 done (100.00%)
+    0:06 [25 rows, 0B] [4 rows/s, 0B/s]
+
+Once you are done with your exploration, enter the ``quit`` command.
+
+Alternatively, you can use the Trino CLI installed directly on your workstation.
+The default server URL in the CLI of http://localhost:8080 matches the port used
+in the command to start the container. More information about using the CLI can
+be found in :doc:`/client/cli`. You can also connect with any other client
+application, such as `DBeaver <https://dbeaver.io/>`_, using
+the :doc:`/client/jdbc`.
+
+If you try to execute a query and see an error message like
```

Comment:
I would get rid of this paragraph and more until "configuring Trino"

Reply:
I removed it, except for one sentence that the status initially shows `(health: starting)`, which I moved above when the `docker ps` command is first introduced.